cmd/gapit/trace.go

```diff
@@ -227,25 +227,26 @@ func (verb *traceVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 	}
 
 	options := &service.TraceOptions{
-		Type:                      api.traceType,
```

Comment:
Many lines flagged as different by github, but I can't see the diff? maybe whitespaces?

Reply:
It is whitespace, gofmt aligns everything and apparently github doesn't highlight whitespace changes.